test/functional/Java11andUp/playlist.xml

```diff
@@ -89,6 +89,45 @@
 			-excludegroups $(DEFAULT_EXCLUDE); \
 			$(TEST_STATUS)
 		</command>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>SE110</subset>
+		</subsets>
+   </test>
+
+
+	<!--
+		Following test covers scenario for both type of constant dynamics (Object and Primitives)
+		Variation 1:
+			This variation runs test in completely interpreted mode.
+		Variation 2:
+			This variation runs test under -Xjit:count=1 option which makes sure that ConstantDynamic is resoleved before we compile a method
+			to test JIT's handling of resolved constant dynamic
+		Variation 3:
+			This variation runs test under -Xjit:count=0 option means JIT faces unresolved constant dynamic and it needs to resolve them before
+			using it.
+		-->
+	<test>
+		<testCaseName>CondyPrimitive</testCaseName>
+		<variations>
+			<variation>'-Xint'</variation>
+			<variation>'-Xjit:count=1,disableAsyncCompilation'</variation>
+			<variation>'-Xjit:count=0,disableAsyncCompilation'</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+			-Xverify:none \
```

Comment:
This is a temporary option until the verifier changes for condy have been merged...  which I thought had already occurred.  Can you confirm this is still required with the latest in `master`?

Reply:
Verified with master repo. We do not need it. Removed in https://github.com/eclipse/openj9/pull/2870/commits/2d8f421c55cf7ae39bb68c90bac24b683061fe29